src/containers/UserEdit.jsx

```diff
@@ -78,7 +77,7 @@ class UserEdit extends React.Component {
     }
 
     let userFields = {}
-    if (!this.props.authType === 'login' || this.props.authType === 'signup') {
+    if (!this.props.authType || this.props.authType === 'signup') {
```

Comment:
This seems more easily expressed as `this.props.authType !== 'login'` (and below

Reply:
Duh!